staged_dependencies.yaml

```diff
@@ -3,6 +3,8 @@ current_repo:
   repo: pharmaverse/admiraltemplate
   host: https://github.com
 upstream_repos:
+  - repo: pharmaverse/admiral.test
+    host: https://github.com
   - repo: pharmaverse/admiral
     host: https://github.com
```

Comment:
Actually admiral is a downstram repo to admiraldev.

Reply:
good spot Thomas - now removed